aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -215,6 +216,11 @@
    */
   public static final String CLIENT_ASSUME_ROLE_REGION = "client.assume-role.region";
 
+  /**
+   * S3 Tag prefix used by {@link S3FileIO}
```

Comment:
can be more descriptive, we need to at least say this is used to tag objects when writing. Also would be great to provide:
1. an example of how to set write tags using prefix
2. a link to S3 tagging documentation

Reply:
Thanks, I have made the changes.

Reply:
Thanks, I have made the changes.